config/config.yaml

```diff
@@ -52,6 +52,9 @@ branch-protection:
       allow_force_pushes: false
       # Disallow deletion of the protected branch.
       allow_deletions: false
+      
+      required_pull_request_reviews:
+        required_approving_review_count: 0
```

Comment:
Add a comment, explaining why we set this to 0.
Also explain why it is necessary to use this setting in addition to "enforce_admins" which is already documented as:
> Prevent administrators from accidentally pushing directly to protected branches